applications/fishing-map/src/features/report/ReportLayerPanel.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react'
+import cx from 'classnames'
+import { useTranslation } from 'react-i18next'
+import { TagList, Tooltip } from '@globalfishingwatch/ui-components'
+import { TagItem } from '@globalfishingwatch/ui-components/dist/tag-list'
+import { DatasetTypes } from '@globalfishingwatch/api-types'
+import { UrlDataviewInstance } from 'types'
+import styles from 'features/workspace/shared/LayerPanel.module.css'
+import { getFlagsByIds } from 'utils/flags'
+import { getSchemaFieldsSelectedInDataview } from 'features/datasets/datasets.utils'
+import {
+  getSourcesSelectedInDataview,
+  isFishingDataview,
+  isPresenceDataview,
+} from 'features/workspace/heatmaps/heatmaps.utils'
+
+type LayerPanelProps = {
+  index: number
+  dataview: UrlDataviewInstance
+}
+
+function ReportLayerPanel({ dataview, index }: LayerPanelProps): React.ReactElement {
+  const { t } = useTranslation()
+  const sourcesOptions: TagItem[] = getSourcesSelectedInDataview(dataview)
```

Comment:
What do you thing about extracting this to shared components ? so we could reuse in the sidebar layer panel and here too. We can go as we are now, but maybe in the UI review I'd go for it

Reply:
I like the idea @j8seangel ! You are referring to shared components in this folder `applications/fishing-map/src/features/workspace/shared/`, right?